adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +106,62 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    if (adConfiguration.childDirectedTreatment) {
```

Comment:
If `adConfiguration.childDirectedTreatment` is `nil`, it means the publisher hasn't specified child directed treatment, which means we need to pass `-1` to Pangle SDK.

Reply:
update this invoke way `[GADMediationAdapterPangle setCoppa:(adConfiguration.childDirectedTreatment ? adConfiguration.childDirectedTreatment.integerValue : -1)];`